composer.json

```diff
@@ -26,7 +26,7 @@
     },
     "bin": ["bin/satis"],
     "require": {
-        "php": "^7.2||^8.0",
+        "php": ">=7.3||^8.0",
```

Comment:
Should probably be kept as `^7.3||^8.0`, otherwise mentioning 8.0 would be redundant.

Reply:
Fair point! Pushing an update now